src/test/java/Homework21.java

```diff
@@ -0,0 +1,41 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework21 extends BaseTest {
+    @Test
+    public void playSong() {
+        //Login
+        provideEmail("pearl.estrada@testpro.io");
+        providePassword("April969!!");
+        clickLoginBtn();
+        //Choose all songs List
+        chooseAllSongsList();
+        //Right click first song
+        contextClickFirstSong();
+        //Choose play option
+        choosePlayOption();
+        //check to see if song is playing
+        Assert.assertTrue(isPlaying());
+    }
+
+    public void chooseAllSongsList() {
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("li a.songs"))).click();
+    }
+
+    public void contextClickFirstSong() {
+        WebElement firstSongElement = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(".all-songs tr.song-item:nth-child(1)")));
+        actions.contextClick(firstSongElement).perform();
+    }
+
+    public void choosePlayOption() {
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("li.playback"))).click();
+    }
+
+    public boolean isPlaying() {
+    WebElement soundBarVisualizer = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("[data-testid='sound-bar-play']")));
+    return soundBarVisualizer.isDisplayed();
+    }
```

Comment:
Page Object Model (POM): recommended to associate the methods with the corresponding page class where they are relevant. In the POM, each page or section of a web application is represented by a separate class. These classes encapsulate the elements and actions specific to that page or section.

Move these methods to the corresponding class according to the POM.

For example
![Screenshot_2](https://github.com/YuriiLyndiuk/A57-QA-Automation-Framework/assets/104761482/b07cd7c1-7977-4e53-80ea-8045ac17f779)